docs/manage/users.rst

```diff
@@ -226,6 +226,14 @@ user and group on an agent can be configured in the ``master.yaml`` file located
        group: root
        gid: 0
 
+.. note::
+
+   A writable ``HOME`` directory is required by all Determined tasks. Out of the box, all official
+   Determined images contain a tool called ``libnss_determined`` that injects users into the
+   container at runtime. If you are building custom images from a base image other than one of
+   Determined's images, you may need to either include ``libnss_determined`` in your custom image or
+   find an equivalent solution for injecting users into the container at runtime.
+
```

Comment:
.. note::

   A writable ``HOME`` directory is required by all Determined tasks. By default, all official Determined images contain a tool called ``libnss_determined`` that injects users into the container at runtime. If you are building custom images using a base image other than those provided by Determined, you may need to take one of the following steps:
- include ``libnss_determined`` in your custom image to ensure user injection works as expected, or 
- find an alternative solution that serves the same purpose of injecting users into the container during runtime for your custom image.

Reply:
you are actually reviewing code from https://github.com/determined-ai/determined/pull/8347 so I'll make this change there.